src/main/java/io/confluent/connect/hdfs/parquet/ParquetHiveUtil.java

```diff
@@ -69,6 +70,21 @@ private Table constructParquetTable(String database, String tableName, Schema sc
     }
     // convert copycat schema schema to Hive columns
     List<FieldSchema> columns = HiveSchemaConverter.convertSchema(schema);
+    List<FieldSchema> partitionFields = partitioner.partitionFields();
+
```

Comment:
Sorry I am confused, it seems that Hive didn't check whether there are overlaps between columns and partition columns. Did you see any error if there is overlap? Is this only applies to parquet or to Avro as well?